run_mpas_analysis

```diff
@@ -71,13 +71,16 @@ def build_analysis_list(config):  # {{{
     oceanTimeSeriesTask = MpasTimeSeriesTask(config=config,
                                              componentName='ocean')
     analyses.append(oceanClimatolgyTask)
+    analyses.append(ocean.StreamfunctionMOC(config, oceanClimatolgyTask))
     analyses.append(ocean.ClimatologyMapMLD(config, oceanClimatolgyTask))
```

Comment:
Moving the MOC to be the first task to run is obviously unrelated to the rest of this PR.  It's just something I noticed when I was running on Edison and the batch job was stuck on MOC with everything else done.

Reply:
I guess the only 'visible' consequence of this is that now the first image on the webpage (template for Ocean diagnostics) has become the MOC plot, but that is OK.

Reply:
it does not bother me.

Reply:
Nope, I much prefer MLD as the "preview" for the ocean webpage so I'm going to move MOC back.  I'll try to think of how this might be addressed (i.e. a different order for tasks on the web page than how they get run.)